v2/book/examples/Pinecone.TypedPath.html

```
@@ -0,0 +1,69 @@
+
+<h1>RSCG nr 284 :  Pinecone.TypedPath</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)